connector/src/main/scala/com/google/cloud/spark/bigquery/BigQueryUtil.scala

```diff
@@ -47,6 +48,28 @@ object BigQueryUtil {
   val QUALIFIED_TABLE_REGEX: Regex =
     s"""^((($PROJECT_PATTERN)[:.])?($DATASET_PATTERN)\\.)?($TABLE_PATTERN)$$""".r
 
+  private def dtoTableSchemaToBqSchema(dtoSchema: TableSchema): Schema = {
+    val fromPbMethod =
+      classOf[Schema]
+        .getDeclaredMethods
+        .toIterable
+        .find(method => method.getName == "fromPb")
+        .get
+
+    fromPbMethod.setAccessible(true)
+    fromPbMethod.invoke(null, dtoSchema).asInstanceOf[Schema]
```

Comment:
why is this null?

Reply:
the first argument is an object this method is invoked from
as far as `fromPb` is a static method, this argument will be ignored
from javadoc: `If the underlying method is static, then the specified {@code obj} argument is ignored. It may be null.`